lib/csv.rb

```diff
@@ -1342,8 +1341,25 @@ def build_header_fields_converter
       accept_nil: true,
     }
     options = @base_fields_converter_options.merge(specific_options)
+    parse_converters(@initial_header_converters, options)
+  end
+
+  def writer_fields_converter
+    @writer_fields_converter ||= build_writer_fields_converter
+  end
+
+  def build_writer_fields_converter
+    specific_options = {
+      builtin_converters: Converters
```

Comment:
It's strange. `Converters` converts `String` to `Integer` or something.
We need to convert anything to `String` on write.

Reply:
I was a bit confused about this one. So I completely removed the builtin_converters